tenant/config/samples/tenancy_v1alpha1_tenant.yaml

```diff
@@ -7,3 +7,7 @@ metadata:
 spec:
   # Add fields here
   tenantAdminNamespaceName: "tenant1admin"
+  tenantAdmins:
+    - kind: ServiceAccount
+      name: t1-user1
+      namespace: default
```

Comment:
So tenantAdmins here can be groups as well? Just double checking.

Reply:
Yes. I have not tested it though.